tests/keysignature_tests.ts

```diff
@@ -52,7 +48,6 @@ function parser(): void {
   catchError('b');
   catchError('Kb');
   catchError('Fb');
-  catchError('Ab');
```

Comment:
Is this removal intentional?

Reply:
Thanks for the review! Yes, it is intentional.

The previous catchError() method would do nothing if there was no exception thrown. When I changed it to QUnit's `throws()`, it is now asserting that an exception WILL be thrown. If the line of code succeeds, the assertion fails.

`Flow.keySignature('Ab')` is a valid key signature. It also appears 7 lines down, and runs without throwing any exception.